src/arrayMethodSort.js

```diff
@@ -4,9 +4,30 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) => `${a}` > `${b}`) {
```

Comment:
this is wrong )
the `.sort` callback expected output is a number `-1 | 0 | 1`, right?
this would not return the expected outputs
```
`${a}` > `${b}`
```
introduce a proper separate helper function:
```
function defaultSort(...) {...}// returns `-1 | 0 | 1`

// then use it as default
function(compareFunction = defaultSort) {...}
```

even if you'd need to compare strings, why not just use `single quotes` or even better - a `String` class:
`'a' > 'b'` || `String(a) - String(b)`

Reply:
single quotes comparison doesn't seem to work. Throws an eslint line `Unexpected constant condition.eslint[no-constant-condition]`